packages/contracts/contracts/L1/ValidatorPool.sol

```diff
@@ -237,6 +237,19 @@ contract ValidatorPool is ReentrancyGuardUpgradeable, ISemver {
         require(success, "ValidatorPool: ETH transfer failed");
     }
 
+    /**
+     * @notice Withdraw a given amount.
+     *
+     * @param _to     Address to withdraw asset to.
+     * @param _amount Amount to withdraw.
+     */
+    function withdrawTo(address _to, uint256 _amount) external nonReentrant {
+        _decreaseBalance(msg.sender, _amount);
+
+        bool success = SafeCall.call(_to, gasleft(), _amount, "");
+        require(success, "ValidatorPool: ETH transfer failed");
+    }
```

Comment:
@kroma-network/l2-protocol What do you think? Is it good to prevent withdrawing to zero address? I actually thought this, but I didn't add because it is not required here.

Reply:
good to have

Reply:
Ok, I'll add it.